README.md

```diff
@@ -27,6 +21,18 @@ If you prefer to do it straight from CLI here is a one-liner for you:
 ```bash
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
 ```
+#### From source
+You can easily build OpenCompose tool from source. All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
+
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it properly:
```

Comment:
Bit harsh saying that it's "built properly" this way.

IMO, there should be two sections:

```
### From source
go get -u foobar

### Manual installation
curl commands, blah blah
```

Saying that `go get` isn't suitable and that you *HAVE* to do make install doesn't seem fitting.